Support construction/destruction patterns of modern debugs()

... which use an implicit "__FUNCTION__:" scope/prefix.

Disclaimer: This patch is untested.

=== modified file 'scripts/find-alive.pl'
--- scripts/find-alive.pl	2016-06-13 23:25:45 +0000
+++ scripts/find-alive.pl	2017-01-22 03:38:41 +0000
@@ -58,42 +58,42 @@ my %Pairs = (
 		'cbdataInternalAlloc: Allocating (\S+)',
 		'cbdataRealFree: Freeing (\S+)',
 	],
 	FD => [
 		'fd_open.*\sFD (\d+)',
 		'fd_close\s+FD (\d+)',
 	],
 	IpcStoreMapEntry => [
 		'StoreMap.* opened .*entry (\d+) for \S+ (\S+)',
 		'StoreMap.* closed .*entry (\d+) for \S+ (\S+)',
 	],
 	sh_page => [
 		'PageStack.* pop: (sh_page\S+) at',
 		'PageStack.* push: (sh_page\S+) at',
 	],
 );
 
 if (!$Pairs{$Thing}) {
     warn("guessing construction/destruction pattern for $Thing\n");
     $Pairs{$Thing} = [
-		"\\b$Thing construct.*, this=(\\S+)",
-		"\\b$Thing destruct.*, this=(\\S+)",
+		"\\b$Thing:? construct.*, this=(\\S+)",
+		"\\b$Thing:? destruct.*, this=(\\S+)",
 	];
 }
 
 die("unsupported Thing, stopped") unless $Pairs{$Thing};
 
 my $reConstructor = $Pairs{$Thing}->[0];
 my $reDestructor = $Pairs{$Thing}->[1];
 
 my %AliveCount = ();
 my %AliveImage = ();
 my $Count = 0;
 while (<STDIN>) {
 	if (my @conIds = (/$reConstructor/)) {
 		my $id = join(':', @conIds);
 		#die($_) if $Alive{$id};
 		$AliveImage{$id} = $_;
 		++$Count unless $AliveCount{$id}++;
 	} 
 	elsif (my @deIds = (/$reDestructor/)) {
 		my $id = join(':', @deIds);

